client/state/login/selectors.js

```diff
@@ -98,7 +92,7 @@ export const isRequestingTwoFactorAuth = ( state ) => {
  * @returns {?string}         Error for the request.
  */
 export const getTwoFactorAuthRequestError = ( state ) => {
-	return get( state, 'login.twoFactorAuthRequestError', null );
+	return state.login.twoFactorAuthRequestError ?? null;
```

Comment:
Also shouldn't need a fallback. The state is either `null` or an `error` object. The only case where the fallback would kick in is if the `TWO_FACTOR_AUTHENTICATION_LOGIN_REQUEST_FAILURE` action object had `undefined` `action.error` property.

Reply:
Good point. If that situation is relevant and it should indeed be a `null`, then that should be handled once by the reducer, not every time by the selector.